src/System.Text.Encoding.CodePages/tests/EncodingCodePages.cs

```diff
@@ -629,6 +633,34 @@ public static void TestRegister1252()
             Assert.Equal(s1252Result, win1252.GetString(enc));
         }
 
+        // This test is run as part of the codepage encodings and EncoderNLS/DecoderNLS tests, since it modifies global
+        // state which they depend on. It is also expected to return an encoding which is used within the calling tests.
```

Comment:
> since it modifies global state which they depend on

What state does it modify?  Where?

Reply:
It doesn't! That was a leftover comment from a copy/paste and this method is now removed.